### ISSUE TITLE
The performance of the best model is lower than the results in the paper?

### ISSUE THREAD

**Issue by shadowclouds** [Alvin-Zeng/PGCN#8]
Thanks for your excellent work.
I trained the model that you provided and found that the best model's(at the epoch 15) performance is  
| IoU thresh | 0.10      | 0.20    | 0.30     | 0.40     | 0.50     | 0.60     | 0.70     | 0.80     | 0.90    | Average |
| mean AP    | 0.6574 | 0.6382 | 0.6009 | 0.5374 | 0.4578 | 0.3369 | 0.2172 | 0.0903 | 0.0134 | 0.3944  |
+------------+--------+--------+--------+--------+--------+--------+--------+--------+--------+---------
And it's lower than results in the paper. Could you provide the pretrained model or explain why this happened ?
Thank you

**Comment by Alvin-Zeng** [comment]
The lowest loss does not lead to the best mAP. This is an open question in both object detection and action localization since we minimize the classification loss rather than mAP. Please test other models and you may get better results.

**Comment by shadowclouds** [comment]
Sorry for excuse. I tried several models but still could not reach the result in your paper. Could you provide your pretrained model and training details?
my email is: chenyunze2018@ia.ac.cn
Thanks for your excellent work!

**Comment by Alvin-Zeng** [comment]
Hi, we have uploaded the trained models and results.
